langkit/templates/pkg_init_spec_ada.mako

```diff
@@ -2,13 +2,9 @@
 
 --  This package provides the library initialization routines
 
-package ${ada_lib_name}.Init is
-
-   procedure Initialize
-     with Export        => True,
-          Convention    => C,
-          External_Name =>
-             "${ada_lib_name.lower()}_initialize";
-   ${ada_doc('langkit.initialize', 3)}
+private package ${ada_lib_name}.Init is
+
+   procedure Initialize;
```

Comment:
I'm wondering, is there really a need to have a separate package (with a fake procedure in the spec) just for a check for Iconv? Why not just inline that in `Implementation` or `Analysis` or `Common` or whatever else needs a body ?

Reply:
We could move that to Implementation or Common, indeed (Analysis may not be generated nowadays). I’ll do that in another commit, thanks.